index.html

```diff
@@ -2656,18 +2640,6 @@ <h2>The <dfn>NFCWatchMode</dfn> enum</h2>
             Return <var>promise</var> and continue the following steps
             <a>in parallel</a>.
           </li>
-          <li>
-            If any exception occurs while running these steps, reject
```

Comment:
Same as above, let's remove custom handling of secure context and remove only following blocks:

```
-          <li>
-            If the <a>incumbent settings object</a> is not a
-            <a>secure context</a>, reject <var>promise</var> with
-            <code>"<a>SecurityError</a>"</code> and abort these steps.
-            <div class="note">
-              Browsers may ignore this rule for development purposes only.
-            </div>
-          </li>
```